crates/node/src/catchup/blobs.rs

```diff
@@ -0,0 +1,71 @@
+use core::mem::take;
+
+use calimero_blobstore::CHUNK_SIZE as BLOB_CHUNK_SIZE;
+use calimero_network::stream::{Message, Stream, MAX_MESSAGE_SIZE as MAX_STREAM_MESSAGE_SIZE};
+use eyre::{bail, Result as EyreResult};
+use futures_util::SinkExt;
+use serde_json::to_vec as to_json_vec;
+
+use crate::types::{CatchupApplicationBlobChunk, CatchupStreamMessage};
+
+pub struct ApplicationBlobChunkSender {
+    batch_size: u8,
```

Comment:
if it'll always be converted to a `usize` to be used, maybe just leave it as that?